README.md

```diff
@@ -1,77 +1,154 @@
+Each row represents either a bam or a cram file along with its corresponding index file. Ensure that all input files have consistent file extensions.
 
-```bash
-nextflow run nf-core/phaseimpute -profile test,<docker/singularity/conda/institute>
-```
+For certain tools and steps within the pipeline, you will also need to provide a samplesheet for the reference panel. Here's an example of what a final samplesheet for a reference panel might look like, covering three chromosomes:
 
-> Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
+```csv
+panel,chr,vcf,index
+Phase3,1,ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz,ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz.csi
+Phase3,2,ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz,ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz.csi
+Phase3,3,ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz,ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz.csi
+```
 
-iv. Start running your own analysis!
+## Running the pipeline
 
-<!-- TODO nf-core: Update the default command above used to run the pipeline -->
+Execute the pipeline with the following command:
 
 ```bash
-nextflow run nf-core/phaseimpute -profile <docker/singularity/conda/institute> --reads '*_R{1,2}.fastq.gz' --genome GRCh37
+nextflow run nf-core/phaseimpute \
+   -profile <docker/singularity/.../institute> \
+   --input <samplesheet.csv>  \
+   --genome "GRCh38" \
+   --panel <phased_reference_panel.csv> \
+   --steps "panelprep,impute" \
+   --tools "glimpse1" \
+   --outdir <OUTDIR>
 ```
 
-See [usage docs](docs/usage.md) for all of the available options when running the pipeline.
+> [!WARNING]
+> Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_; see [docs](https://nf-co.re/docs/usage/getting_started/configuration#custom-configuration-files).
+
+For more details and further functionality, please refer to the [usage documentation](https://nf-co.re/phaseimpute/usage) and the [parameter documentation](https://nf-co.re/phaseimpute/parameters).
 
-## Documentation
+## Description of the different steps of the pipeline
 
-The nf-core/phaseimpute pipeline comes with documentation about the pipeline, found in the `docs/` directory:
+Here is a short description of the different steps of the pipeline.
+For more information please refer to the [documentation](https://nf-core.github.io/phaseimpute/usage/).
```

Comment:
[documentation](https://nf-core.github.io/phaseimpute/usage/) soems not to work

The current link does not work
```suggestion
For more information please refer to the [usage documentation](https://nf-co.re/phaseimpute/usage).
```

Reply:
Done in #152